utils/_helpers.scss

```diff
@@ -68,39 +68,10 @@ $debug: false !default;
 @mixin user-select($value: none) {
   @include prefixer(user-select, $value, webkit moz ms spec);
 }
-@mixin media($query, $total-columns: $grid-columns) {
-  @if length($query) == 1 {
-    @media screen and ($default-feature: nth($query, 1)) {
-      $default-grid-columns: $grid-columns;
-      $grid-columns: $total-columns !global;
-      @content;
-      $grid-columns: $default-grid-columns !global;
-    }
-  } @else {
-    $loop-to: length($query);
-    $media-query: "screen and ";
-    $default-grid-columns: $grid-columns;
-    $grid-columns: $total-columns !global;
-
-    @if is-not(is-even(length($query))) {
-      $grid-columns: nth($query, $loop-to) !global;
-      $loop-to: $loop-to - 1;
-    }
-
-    $i: 1;
-    @while $i <= $loop-to {
-      $media-query: $media-query + "(" + nth($query, $i) + ": " + nth($query, $i + 1) + ") ";
-
-      @if ($i + 1) < $loop-to {
-        $media-query: $media-query + "and ";
-      }
-
-      $i: $i + 2;
-    }
-    @media #{$media-query} {
-      @content;
-      $grid-columns: $default-grid-columns !global;
-    }
+// @todo
+@mixin media($query) {
```

Comment:
Would be great to have a temporarily solution that makes components runs well
<img width="724" alt="screen shot 2017-03-31 at 14 09 48" src="https://cloud.githubusercontent.com/assets/1137569/24550171/3be4a276-161d-11e7-861b-8160ee9bb6af.png">

ex.
```
$breakpoints: (
  small: 767px,
  medium: 992px,
  large:  1200px
) !default;
```

```
@mixin media($breakpoint) {
  // If the key exists in the map
  @if map-has-key($breakpoints, $breakpoint) {
    // Prints a media query based on the value
    @media (min-width: map-get($breakpoints, $breakpoint)) {
      @content;
    }
  }
 
  // If the key doesn't exist in the map
  @else {
    @warn "Unfortunately, no value could be retrieved from `#{$breakpoint}`. "
        + "Available breakpoints are: #{map-keys($breakpoints)}.";
  }
}
```